wails-frontend/src/pages/importFile.tsx

```diff
@@ -48,19 +50,37 @@ const ImportFile = () => {
   };
 
   return (
-    <section>
-      <div>{req.Msg}</div>
-      <div className="baseline">{baselineStr()}</div>
-      <div className="flex">
-        <button className="btn" onClick={handleCancel}>
-          Cancel
-        </button>
-        <button className="btn" onClick={handleApply}>
-          {applyStr()}
-        </button>
+    <div className="bg-primary flex flex-col justify-center items-center h-screen w-full">
+      <div className="w-1/4  max-w-sm  min-w-fit">
```

Comment:
I don't think we need to use **max-w-sm** here.

Reply:
uhmmm I think we do because the button will be too stretched if we don't have a max-w :) I can attach screen in this conv if necessary